audio.py

```diff
@@ -52,3 +52,9 @@ def glue_tracks(tracks):
     for audio, fade in tracks[1:]:
         final = final.append(audio, crossfade=fade)
     return final
+
+def normalize_audio(podcast_file):
+    """
+    This function normalize track
+    """
+    return normalize(podcast_file, headroom=-1.5)
```

Comment:
What's the reason to use `-1.5` for the `headroom` parameter? _trial & error_?

Reply:
I did a lot of test and -1.5 was the best setting to our kind of audio (Audacity set by default -1.0). Yes, trial & error.